src/components/Header.js

```diff
@@ -1,10 +1,30 @@
 import React from "react";
+import AppBar from '@mui/material/AppBar';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
+import MenuIcon from '@mui/icons-material/Menu';
 
 const Header = (props) => {
   return (
-    <header className="App-header">
-      <h2>{props.text}</h2>
-    </header>
+    <AppBar position="static">
+    <Toolbar>
+      <IconButton
+        size="large"
+        edge="start"
+        color="inherit"
+        aria-label="menu"
+        sx={{ mr: 2 }}
+      >
+        <MenuIcon />
+      </IconButton>
+      <Typography variant="h4" component="div" sx={{ flexGrow: 1 }}>
+        Lucho TV
+      </Typography>
+      <Button color="inherit">Iniciar Sesion</Button>
```

Comment:
Remove this, we don't have sessions or accounts in this app

Reply:
Removed!